spec/support/factory_girl.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
So if you're including this, you should be able to remove `FactoryGirl` from your `build` statements, so it can look like `build(:education)`. It's cleaner as well, so I recommend doing it.